text/0999-make-hash-built-in.md

```diff
@@ -30,55 +27,42 @@ project-link: Leave as is
 suite: Leave as is
 -->
 
-# <RFC title>
+# Make `(hash)` a built in helper 
 
 ## Summary
 
-> One paragraph explanation of the feature.
+Today, when using gjs/gts/`<template>`, in order to bind event listeners, folks _must import_ the `(hash)` helper.
```

Comment:
Certainly a copy-paste but `in order to bind event listeners` does not make sense in this document.